include/xtensor/xaccumulator.hpp

```diff
@@ -200,10 +210,11 @@ namespace xt
             using init_type = typename F::init_value_type;
             using accumulate_functor_type = typename F::accumulate_functor_type;
             using expr_value_type = typename std::decay_t<E>::value_type;
-            //using return_type = std::conditional_t<std::is_same<init_type, void>::value, typename std::decay_t<E>::value_type, init_type>;
+            // using return_type = std::conditional_t<std::is_same<init_type, void>::value, typename
+            // std::decay_t<E>::value_type, init_type>;
 
-            using return_type = std::decay_t<decltype(std::declval<accumulate_functor_type>()(std::declval<init_type>(),
-                                                                                              std::declval<expr_value_type>()))>;
+            using return_type = std::decay_t<decltype(std::declval<accumulate_functor_type>(
+            )(std::declval<init_type>(), std::declval<expr_value_type>()))>;
```

Comment:
Looks weird, I don't know if something could be done

Reply:
I also don't like it. Clang-format seems to have trouble with chained calls with 0 or1 parameter.
It also has very little option for how to break `<>`.

I could not find anything that makes it look good, without sacrificing other thinks (like not breaking assignments).

Reply:
We could use more typedefs, or `// clang-format: off`

Reply:
https://github.com/xtensor-stack/xtensor/issues/2633